source/tutorials/nix-language.md

```diff
@@ -129,6 +129,8 @@ nix-repl> { a.b.c = 1; }
 nix-repl> :p { a.b.c = 1; }
 { a = { b = { c = 1; }; }; }
 ```
+
+to exit [`nix repl`][nix-repl] use `:q`
```

Comment:
Also this line should go *above* the note, not inside it. It has nothing to do with the note.

```suggestion

Type `:q` to exit [`nix repl`][nix-repl].

```